tutor/specs/e2e/helpers.ts

```diff
@@ -108,9 +108,9 @@ export const selectCalendarSidebarOption = async (page: Page, option: string) =>
     await page.click(`.add-assignment-sidebar >> text="${option}"`)
 }
 
-export const signTerm = async (page: Page, termId: string) => {
-    await page.evaluate(() => {
-        const term = (window as any)._MODELS?.user.terms.get(termId)
+export const signTerm = async (page: Page, _termId: string) => {
+    await page.evaluate((_termId) => {
+        const term = (window as any)._MODELS?.user.terms.get(_termId)
```

Comment:
don't you have to pass termId to the second arg of `evaluate`?

Reply:
Oh that's right the new runner does need that. Good catch! Updated: https://github.com/openstax/tutor-js/pull/3750/commits/1ad8abc9320b4e095612696417a91f5a30b46e1c

Reply:
yea those evaluates are hard because both typescript and my brain can't remember that those lines of code will be yoinked into a different runtime context